tests/Functional/EmbeddedMappingTest.php

```diff
@@ -50,7 +50,7 @@ public function testCreateDocumentWithEmbedded(): void
         $form[$attributeAddressStreet] = 'A wonderful street to live';
 
         $crawler->filter('.field-container .sonata-ba-action[title="Add new"]')->click();
-        $crawler = $this->client->waitFor('.phone_number_number');
+        // $crawler = $this->client->waitFor('.phone_number_number');
```

Comment:
this line can be reverted now and after https://github.com/sonata-project/form-extensions/pull/292 it should work fine

Reply:
Done